packages/profiler/src/main.rs

```diff
@@ -71,7 +71,7 @@ fn main() {
 }
 
 // Pretty much stolen from `/contracts/hackatom/tests/integration.rs`
-fn call_things(mut deps: &mut MockInstance) {
+fn call_things(deps: &mut MockInstance) {
```

Comment:
My local linter (v1.53.0) didn't catch those. Must be the work of the v1.57.0 one.